application/src/main/java/bisq/application/DefaultApplicationService.java

```diff
@@ -96,6 +97,11 @@
     private final SocialService socialService;
     private final SecurityService securityService;
 
+    /**
+     * Initialization status, updated as each required service bootstraps
+     */
+    private final Observable<String> initStatus = new Observable<>();
```

Comment:
I guess we will need some more finegrained state info. But I think that should be preovided by the domain services directly, so to have only one state here seems ok.

Reply:
Depending on how granular we want to go, we can add a 2nd line with a more detailed substatus message and icons.

For now the main status is added as an enum.